source/nodejs/adaptivecards/src/host-config.ts

```diff
@@ -3,10 +3,18 @@
 import * as Enums from "./enums";
 import * as Utils from "./utils";
 import * as Shared from "./shared";
+import { HostCapabilities } from "./host-capabilities";
 
-export interface IValidationError {
-    error: Enums.ValidationError,
-    message: string;
+function parseHostConfigEnum(targetEnum: { [s: number]: string }, value: string | number, defaultValue: any): any {
```

Comment:
>any): any [](start = 104, length = 9)

`number` seems to be better choice since this function returns number, so it seems we can assume deafultValue to be number.

Reply:
Fixed: https://github.com/microsoft/AdaptiveCards/pull/3508/commits/6f4fec115d80a095a9051d14fa8d288f4f0b7a14